resources/ecr-scan-check.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python3
+
+import argparse
+import boto3
+import json
+import os
+import sys
+import time
+
+
+def logoutput(level, message):
+    colours = {
+        "Info": "\033[1;34m",
+        "Error": "\033[0;31m",
+        "Success": "\033[0;32m",
+        "end": "\033[0m",
+    }
+
+    string = "{}{}:{} {}".format(colours[level], level, colours["end"], message)
+    print(string)
+
+
+def get_severities(min_severity):
+    severities = ["CRITICAL", "HIGH", "MEDIUM", "LOW", "INFORMATIONAL"]
+
+    if min_severity == "ANY":
```

Comment:
I'd be tempted to ditch the `ANY` as it's not really a log level and instead just give the log level a default of `None`